### PR TITLE
Chunking_introduction notebook improvements

### LINKED ISSUE #48: Update the content of chunking_introduction.ipynb
The goal of this issue is to discuss what we should put in [chunking introduction notebook](https://github.com/pangeo-data/foss4g-2022/blob/main/tutorial/pangeo101/chunking_introduction.ipynb).

This comes from comments and discussions in https://github.com/pangeo-data/foss4g-2022/pull/45.

I believe we should do the following, but this needs discussion (especially with @tinaok):

- [x] Either remove the word compression or talk about it somewhere, a short chapter should be fine. Some subjects to talk about
  - Generalities (why compression matters, how it is a gain in processing time despite the overcost of compression -- loading big files in memory is slow, different algorithms currently used in scientific domains, Python blosc, etc.),
  - A section showing the difference between a NetCDF or zarr file with and witout compression and the resulting size
  - Default of Xarray / Various data formats (is NetCDF compression activated by default?).
  - Compressions is often apply on chunks for optimized access.
- [ ] Add more context about Xarray and Chunking before introducing kerchunk. Begin with `open_dataset` and the `chunks` attribute, introduce Dask Arrays, talk about lazyness and sequential processing. Talk also of native chunks in files here, if possible without kerchunk (with h5py maybe?).
- [ ] Kerchunk part: introduce it by talking about the difficulties of reading a dataset composed of several files in a optimal way. Then explain how it reads file metadata and detect chunks, and construct a Zarr compatible metadatafiles that allows for optimized access of whole dataset with Xarray. Should we keep the main metadata file creation in the notebook? I'm not sure of that.

### PR DESCRIPTION
**Pull request by guillaumeeb**
Here are some modifications I propose on the chunking_introduction notebook.

Closes #48 (even if not all suggestions were implemented).

### CODE REVIEW

File: tutorial/pangeo101/chunking_introduction.ipynb [hunk 1]
```diff
@@ -69,11 +69,13 @@
    "source": [
```
Comment (guillaumeeb) [inline tutorial/pangeo101/chunking_introduction.ipynb:69]: <p>We are still taking of global coverage, is that true?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-data/foss4g-2022/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (annefou) [reply tutorial/pangeo101/chunking_introduction.ipynb:69]: Yes the data is now global. But we only select a small areao of interest.

Comment (guillaumeeb) [reply tutorial/pangeo101/chunking_introduction.ipynb:69]: You mean the data is across all the time dimension?

In my comprehension global means over the entire world.

Comment (annefou) [reply tutorial/pangeo101/chunking_introduction.ipynb:69]: Yes. The data is supposed to be over the entire world. This is why chunking is becoming interesting and necessary.

Comment (guillaumeeb) [reply tutorial/pangeo101/chunking_introduction.ipynb:69]: OK, I understand how you see it.

What I was meaning is that talking about global coverage, but filtering down to a small AOI first thing off is kind of opposite.

But what you're meaning is: "Thanks to Pangeo we can easily open and filter down a global coverage dataset to a small AOI."

Comment (annefou) [reply tutorial/pangeo101/chunking_introduction.ipynb:69]: yes... True it is not super clear...

File: tutorial/pangeo101/chunking_introduction.ipynb [hunk 2]
```diff
@@ -354,300 +369,366 @@
...
-    "collapsed": false,
     "pycharm": {
-     "name": "#%%\n"
+     "name": "#%% md\n"
     }
-   }
+   },
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "    <i class=\"fa-check-circle fa\" style=\"font-size: 22px;color:#666;\"></i> <b>Attention</b>\n",
```
Comment (annefou) [inline tutorial/pangeo101/chunking_introduction.ipynb:392]:
```suggestion
    "    <i class=\"fa-check-circle fa\" style=\"font-size: 22px;color:#666;\"></i> <b>Warning</b>\n",
```

**Review by annefou — APPROVED** [review]

File: tutorial/pangeo101/chunking_introduction.ipynb (hunk 2 above)
Comment (guillaumeeb) [reply tutorial/pangeo101/chunking_introduction.ipynb]: 👍

**Pull request merged**